docs/ops/activation/GELU_7.md

```diff
@@ -2,53 +2,63 @@
 
 **Versioned name**: *Gelu-7*
 
-**Category**: *Activation*
+**Category**: *Activation function*
 
-**Short description**: Calculates Gaussian error linear.
+**Short description**: Gaussian error linear unit element-wise activation function.
 
-**Detailed description**: `Gelu(x) = x * Φ(x)`, where `Φ(x)` is the Cumulative Distribution Function for Gaussian Distribution.
-The Gelu operation is introduced in the [paper](https://arxiv.org/abs/1606.08415).
+**Detailed description**:
 
-**Attributes**
+*Gelu* operation is introduced in this [article](https://arxiv.org/abs/1606.08415).
+It performs element-wise activation function on a given input tensor, based on the following mathematical formula:
 
-* *approximation_mode*
+\f[
+    Gelu(x) = x\cdot\Phi(x)
+\f]
 
-  * **Description**: Specifies the formulae to calculate the output.
-  * **Range of values**:
-    * `erf` -- calculate output using the Gauss error function.
-    * `tanh` -- calculate output using tanh approximation
-  * **Type**: `string`
-  * **Default value**: `erf`
-  * **Required**: *no*
+where `Φ(x)` is the Cumulative Distribution Function for Gaussian Distribution.
 
+The *Gelu* function may be approximated in two different ways based on *approximation_mode* attribute.
 
-**Mathematical Formulation**
+For `erf` approximation mode, *Gelu* function is represented as:
 
-For the `erf` approximation mode:
 \f[
-    Gelu(x) = 0.5 \cdot x \cdot (1.0 + erf((x) / \sqrt{2})
+    Gelu(x) = x\cdot\Phi(x) = x\cdot\frac{1}{2}\cdot\left[1 + erf\left(x/\sqrt{2}\right)\right]
 \f]
 
-For the `tanh` approximation mode:
+For `tanh` approximation mode, *Gelu* function is represented as:
 
 \f[
-    Gelu(x) \approx 0.5 \cdot x \cdot (1.0 + tanh(\sqrt{2.0/pi} \cdot (x + 0.044715 \cdot x ^ 3))
+    Gelu(x) \approx 0.5\cdot x\cdot \left(1 + \tanh\left[\sqrt{2/\pi} \cdot (x + 0.044715 \cdot x^3)\right]\right)
```

Comment:
![image](https://user-images.githubusercontent.com/20084299/116396107-3e745600-a825-11eb-9948-3a7f6a8fb001.png)
I would suggest to unify math formulas to easier see the difference between `erf` and `tanh` modes.
For example: `0.5 * x` ---> `x * 1/2` as in the `erf`.

Reply:
Done

Reply:
As you referred to the [paper](https://arxiv.org/pdf/1606.08415.pdf) where both of the formulas are presented in the previous way, maybe this suggestion was not so relevant. If you prefer don't change it would be okay.